jar-connector/src/main/java/com/sixsq/slipstream/connector/SystemConfigurationParametersFactoryBase.java

```diff
@@ -34,6 +33,20 @@
 
 	protected Map<String, ServiceConfigurationParameter> referenceParameters = new HashMap<String, ServiceConfigurationParameter>();
 
+	public static final String NATIVE_CONTEXTUALIZATION_KEY = "native-contextualization";
+
+	public static final String NATIVE_CONTEXTUALIZATION_NEVER = "never";
+	public static final String NATIVE_CONTEXTUALIZATION_ALWAYS = "always";
+	public static final String NATIVE_CONTEXTUALIZATION_LINUX_ONLY = "linux only";
+	public static final String NATIVE_CONTEXTUALIZATION_WINDOWS_ONLY = "windows only";
```

Comment:
Done.
And I updated the corresponding values in the Client: slipstream/SlipStreamClient@780ba82a6302d9f028096968e3abf7340d2a983a